### PR TITLE
JWKS support

### PR DESCRIPTION
**Pull request by darioAnongba**
JWKS support

### CODE REVIEW

File: pkg/toolkit/app/http/middleware/auth/auth_test.go [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
 	apikeyHeader        = "X-API-Key"
 	authorizationHeader = "Authorization"
```
Comment (6l20) [inline pkg/toolkit/app/http/middleware/auth/auth_test.go:30]: These values could be consts

File: .env.remote [hunk 2]
```diff
@@ -26,6 +26,6 @@ AUTH_API_KEY=with-key
...
...eys":["56202652FDFFD802B7252A456DBD8F3ECC0352BBDE76C23B40AFE8AEBD714E2E","1476c66de79a57e8ab4cadceccbe858c99e5edf3bffea5404b15322b5421e18c"]},{"URLs":["https://user1:secret@besu2.ops.consensys.net"],"privateAddress":["Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs="]},{"URLs":["https://user1:secret@besu3.ops.consensys.net"],"privateAddress":["k2zXEin4Ip/qBGlRkJejnGWdP9cjkK+DAvKNW31L2C8="]}],"quorum":[{"URLs":["https://user1:secret@quorum1.ops.consensys.net"],"privateAddress":["BULeR8JyUWhiuuCMU/HLA0Q5pzkYT+cHII3ZKBey3Bo="],"privateTxManager":{"url":"https://user1:secret@quorum1.ops.consensys.net/tessera","type":"Tessera"},"fundedPublicKeys":["0x6009608a02a7a15fd6689d6dad560c44e9ab61ff"],"fundedPrivateKeys":["86B021CCB810F26A30445B85F71E4C1596A11A97DDF9B9E348AC93D1DA6735BC"]},{"privateAddress":["QfeDAys9MPDs2XHExtc84jKGHxZg/aj52DTh0vtA3Xc="]},{"privateAddress":["1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg="]}],"geth":[{"URLs":["https://user1:secret@geth.ops.consensys.net"],"fundedPublicKeys":["0xdbb881a51CD4023E4400CEF3ef73046743f08da3"],"fundedPrivateKeys":["5FBB50BFF6DFAD35C4A374C9237BA2F7EAED9C6868E0108CB259B62D68029B1A"]}]},"oidcClientID":"","oidcClientSecret":"","oidcTokenURL":""}
```
Comment (6l20) [inline .env.remote:29]: I have seen that issuerUrl is more commonly used

**Review by 6l20 — APPROVED** [review]

File: .env.remote (hunk 2 above)
Comment (darioAnongba) [reply .env.remote:29]: this is the token URL, which is different than the issuer URL passed as ENV var to Orchestrate.

File: pkg/toolkit/app/http/middleware/auth/auth_test.go (hunk 1 above)
Comment (darioAnongba) [reply pkg/toolkit/app/http/middleware/auth/auth_test.go:30]: done

**Pull request merged**